google/api_core/rest_streaming_async.py

```diff
@@ -22,7 +22,7 @@
     import google.auth.aio.transport
 except ImportError as e:  # pragma: NO COVER
     raise ImportError(
-        "google-auth>=2.35.0 is required to use asynchronous rest streaming."
+        "google-api-core[async_rest] is required to use asynchronous rest streaming."
```

Comment:
Shall we update this to:

```
"google-api-core[async_rest] is required to use asynchronous rest streaming." Install google-api-core with async_rest extra using `pip install google-api-core[async_rest]`."
```

Reply:
Fixed in https://github.com/googleapis/python-api-core/pull/701/commits/a8af7912063eeaa09688051da2659a9badab9d08